src/rust/ide/engine-api/build.rs

```diff
@@ -0,0 +1,138 @@
+use ensogl_build_utilities::absolute_path;
+
+use std::fs;
+use std::fs::File;
+use std::fs::create_dir_all;
+use std::io::prelude::*;
+use std::path::PathBuf;
+use std::process::Command;
+use zip_extensions::read::ZipArchiveExtensions;
+
+
+
+// =========================
+// == Hardcoded constants ==
+// =========================
+
+
+/// The path where downloaded artefacts will be stored.
+const ZIP_PATH   : &str = "./pkg/fbs-schema.zip";
+/// The path where downloaded artefacts will be unziped.
+const UNZIP_PATH : &str = "./pkg/fbs-upload/fbs-schema/";
+
```

Comment:
Usually generated files should be in $OUT directory according to https://doc.rust-lang.org/cargo/reference/build-scripts.html ; We don't do this with parser, because of flaw in wasm_bindgen library (it does not allow to put env variables to paths of js snippets).

I'm sure that I would download zip to the $OUT dir, and there wie should extract fbs files. However I'm not sure if I want to generate files there as well - I've seen that IDE does not work well with files generated in $OUT. But you can give it a try.